### DISCUSSION TITLE
umiMax css 低版本兼容问题

### DISCUSSION THREAD

**Discussion by Dienww** [umijs/umi#11756] [Q&A]
碰到了低版本浏览器的兼容问题，参考了官网的低版本兼容配置和 #8658 的解决方案，但 css 在较低版本的浏览器上部分样式无法生效。
排查了打包产物，定位到是样式变量引发的：
![image](https://github.com/umijs/umi/assets/32919473/c43f3c0f-5524-46f6-8cd0-d611ca6d4c50)
请问有什么好的解决方案吗？

**Comment by fz6m** [comment]
如果浏览器的版本低到一定程度，以至于不支持 css 变量，就把 css var 改写成 `var(--name, 20px)` 加上第二个 fallback 值，如果是第三方库，就不要用带 css 变量的库，因为绝大部分浏览器都是支持 css 变量的，只有非常久远的浏览器不支持，在那个时代绝大部分都是手写的。
